extensions/amp-render/1.0/test/test-amp-render.js

```diff
@@ -144,6 +142,57 @@ describes.realWin(
       expect(fetchStub).to.have.been.calledOnce;
     });
 
+    it('should render with layout=container', async () => {
```

Comment:
Please also add test cases for successful and unsuccessful calls to `attemptChangeHeight`

Reply:
Done